emails/payin_bankwire_expired.spt

```diff
@@ -1,11 +1,11 @@
-{{ _("The bank wire to your Liberapay wallet has expired") }}
+{{ _("The initiated bank wire to your Liberapay wallet has expired") }}
```

Comment:
Makes it clearer that it failed on the platform end, and alludes to how it can be fixed.